source/postprocess/volume_of_fluid_statistics.cc

```diff
@@ -0,0 +1,132 @@
+/*
+  Copyright (C) 2011 - 2018 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/postprocess/volume_of_fluid_statistics.h>
+#include <aspect/simulator_access.h>
+#include <aspect/volume_of_fluid/handler.h>
+#include <aspect/global.h>
```

Comment:
the `simulator_access.h` and `global.h` include are already implicitly included via the header file, please remove them from here.